parm/config/gfs/config.atmanl

```diff
@@ -28,6 +29,7 @@ export layout_y_atmanl=@LAYOUT_Y_ATMANL@
 export io_layout_x=@IO_LAYOUT_X@
 export io_layout_y=@IO_LAYOUT_Y@
 
-export JEDIEXE=${EXECgfs}/fv3jedi_var.x
+export VAREXE=${EXECgfs}/fv3jedi_var.x
+export FV3INCEXE=${EXECgfs}/fv3jedi_fv3inc.x
```

Comment:
This could become `JEDIEXE` in `config.atmanlfv3inc`

Reply:
@aerorahul For consistency then, why not define JEDIEXE=${EXECgfs}/fv3jedi_var.x in config.atmanlvar (formerly config.atmanlrun) rather than config.atmanl?